Trying to deploy but failed.

I was trying to deploy this  source into my personal home page, https://hantsy.github.io, but failed in the Github actions.

```bash
Run yes | bin/deploy --verbose  --src source --deploy master
+ shift
+ [[ 4 > 0 ]]
+ key=--src
+ case $key in
+ SRC_BRANCH=source
+ shift
+ shift
+ [[ 2 > 0 ]]
+ key=--deploy
+ case $key in
+ DEPLOY_BRANCH=master
+ shift
+ shift
+ [[ 0 > 0 ]]
+ set -e
+ echo Deploying...
Deploying...
+ echo 'Source branch: source'
Source branch: source
+ echo 'Deploy branch: master'
Deploy branch: master
+ read -r -p 'Do you want to proceed? [y/N] ' response
+ [[ ! y =~ ^([yY][eE][sS]|[yY])+$ ]]
+ git diff-index --quiet HEAD --
+ echo 'Changes to the following files are uncommitted:'
Changes to the following files are uncommitted:
+ git diff-index --name-only HEAD --
bin/cibuild
bin/deploy
+ echo 'Please commit the changes before proceeding.'
+ echo Aborting.
+ [[ bin/deploy = \b\i\n\/\d\e\p\l\o\y ]]
+ exit 1
yes: standard output: Broken pipe
Please commit the changes before proceeding.
Aborting.
Error: Process completed with exit code 1.
````

> Hi @hantsy , curious to know how you resolved this issue. I am facing the same issue, getting the same error as "Changes to the following files are uncommitted:" and the master branch is not being built.
> 
> I also did not fork the repo, rather cloned, edited, and uploaded it to my own repo. Is there any solution where we don't fork the original repo?

My error log, in case that helps:

```
0s
Run yes | bin/deploy --verbose  --src source --deploy master
  yes | bin/deploy --verbose  --src source --deploy master
  shell: /usr/bin/bash -e {0}
+ shift
+ [[ 4 > 0 ]]
+ key=--src
+ case $key in
+ SRC_BRANCH=source
+ shift
+ shift
+ [[ 2 > 0 ]]
+ key=--deploy
+ case $key in
+ DEPLOY_BRANCH=master
+ shift
+ shift
+ [[ 0 > 0 ]]
+ set -e
+ echo Deploying...
+ echo 'Source branch: source'
Deploying...
+ echo 'Deploy branch: master'
Source branch: source
+ read -r -p 'Do you want to proceed? [y/N] ' response
Deploy branch: master
+ [[ ! y =~ ^([yY][eE][sS]|[yY])+$ ]]
+ git diff-index --quiet HEAD --
+ echo 'Changes to the following files are uncommitted:'
Changes to the following files are uncommitted:
+ git diff-index --name-only HEAD --
.bundle/config
+ echo 'Please commit the changes before proceeding.'
Please commit the changes before proceeding.
Aborting.
+ echo Aborting.
+ [[ bin/deploy = \b\i\n\/\d\e\p\l\o\y ]]
+ exit 1
yes: standard output: Broken pipe
Error: Process completed with exit code 1.
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -46,6 +46,6 @@ jobs:
         fi
         echo "::set-output name=DEPLOY_BRANCH::gh-pages"
     - name: Deploy website 
-      run:  yes | bin/deploy --verbose ${{ steps.setup.outputs.NO_PUSH }}
+      run:  yes | bash bin/deploy --verbose ${{ steps.setup.outputs.NO_PUSH }}
                     --src ${{ steps.setup.outputs.SRC_BRANCH }} 
                     --deploy ${{ steps.setup.outputs.DEPLOY_BRANCH }} 

